src/Sylius/Component/Review/Factory/ReviewFactory.php

```diff
@@ -57,8 +59,9 @@ public function createForSubject(ReviewableInterface $subject)
     /**
      * {@inheritdoc}
      */
-    public function createForSubjectWithReviewer(ReviewableInterface $subject, ReviewerInterface $reviewer = null)
+    public function createForSubjectWithReviewer(ReviewableInterface $subject, ?ReviewerInterface $reviewer = null): ReviewInterface
```

Comment:
We can remove the default `null` value here, when called without the second parameter then `createForSubject` should be used instead.